src/Platforms/SqlitePlatform.php

```diff
@@ -328,14 +330,47 @@ protected function _getCommonIntegerTypeDeclarationSQL(array $column)
         return ! empty($column['unsigned']) ? ' UNSIGNED' : '';
     }
 
+    /**
+     * Disables schema emulation.
+     *
+     * Schema emulation is enabled by default to maintain backwards compatibility.
```

Comment:
This comment would make more sense if applied to the `$schemaEmulationEnabled` property. Otherwise, it's unnecessarily duplicated in the following method.

Reply:
Discussed above, only the methods are public API